scrapy/spiders/sitemap.py

```diff
@@ -2,17 +2,17 @@
 import logging
 import six
 
+from six.moves.urllib.parse import urlparse
+
 from scrapy.spiders import Spider
 from scrapy.http import Request, XmlResponse
 from scrapy.utils.sitemap import Sitemap, sitemap_urls_from_robots
 from scrapy.utils.gz import gunzip, gzip_magic_number
 
-
```

Comment:
Please, remove unnecessary changes like this line removal and the other one at line 15.